src/Nethermind/Nethermind.Network/Config/INetworkConfig.cs

```diff
@@ -31,6 +31,9 @@ public interface INetworkConfig : IConfig
         [ConfigItem(Description = "List of nodes for which we will keep the connection on. Static nodes are not counted to the max number of nodes limit.", DefaultValue = "null")]
         string? StaticPeers { get; set; }
 
+        [ConfigItem(Description = "Use tree is available through a DNS name. By keep empty for the default all.{chainName}.ethdisco.net", DefaultValue = "null")]
```

Comment:
Instead of "By keep empty for default all." Say "keep it empty for the default of {chainName}.ethdisco.net"